<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MonitorForm</class>
 <widget class="QDialog" name="MonitorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>600</width>
    <height>500</height>
   </rect>
  </property>
  <property name="name" stdset="0">
   <cstring>Form1</cstring>
  </property>
  <property name="caption" stdset="0">
   <string>SSD Monitor</string>
  </property>
  <widget class="QPushButton" name="btnReset">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>20</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset</string>
   </property>
   <property name="autoDefault">
    <bool>true</bool>
   </property>
   <property name="name" stdset="0">
    <cstring>btnReset</cstring>
   </property>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>20</y>
     <width>461</width>
     <height>91</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">#groupBox {
	border : 1px solid gray;
	border-radius : 9px;
}</string>
   </property>
   <property name="title">
    <string/>
   </property>
   <property name="name" stdset="0">
    <cstring>groupBox1</cstring>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write:</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelWrite</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWriteCmdCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>30</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtWriteCmdCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>10</y>
      <width>60</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Cmd Count</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>10</y>
      <width>91</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Speed [MB/s]</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelSpeed</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWriteSpeed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>30</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtWriteSpeed</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtReadSpeed">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtReadSpeed</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWritePageCount">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>30</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtWritePageCount</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtReadPageCount">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtReadPageCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>370</x>
      <y>10</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Page Count</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelSectorCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>60</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Read:</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelRead</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtReadCmdCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>60</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtReadCmdCount</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>120</y>
     <width>461</width>
     <height>80</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
	font-weight : bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>Garbage Collection</string>
   </property>
   <property name="name" stdset="0">
    <cstring>groupBox2</cstring>
   </property>
   <widget class="QLineEdit" name="txtGCCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtGCCount</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtWriteAmp">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtWriteAmp</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtGCErase">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>40</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtGCErase</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>66</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Call Count</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelGCCount</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Write Amplification</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelWriteAmp</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>375</x>
      <y>20</y>
      <width>55</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Block Erase</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelGCSector</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QGroupBox" name="groupBox">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>210</y>
     <width>341</width>
     <height>70</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">QGroupBox {
    border: 1px solid gray;
    border-radius: 9px;
    margin-top: 0.5em;
	font-weight : bold;
}

QGroupBox::title {
    subcontrol-origin: margin;
    left: 10px;
    padding: 0 3px 0 3px;
}</string>
   </property>
   <property name="title">
    <string>TRIM</string>
   </property>
   <property name="name" stdset="0">
    <cstring>groupBox3</cstring>
   </property>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>20</y>
      <width>56</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Call Count</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelTrimCount</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTrimCount">
    <property name="geometry">
     <rect>
      <x>110</x>
      <y>40</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtTrimCount</cstring>
    </property>
   </widget>
   <widget class="QLineEdit" name="txtTrimEffect">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>40</y>
      <width>89</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>0</string>
    </property>
    <property name="alignment">
     <set>Qt::AlignRight</set>
    </property>
    <property name="name" stdset="0">
     <cstring>txtTrimEffect</cstring>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>20</y>
      <width>100</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>Trimmed LBAs</string>
    </property>
    <property name="name" stdset="0">
     <cstring>txtLabelTrimEffect</cstring>
    </property>
   </widget>
  </widget>
  <widget class="QLineEdit" name="txtSSDUtil">
   <property name="geometry">
    <rect>
     <x>380</x>
     <y>300</y>
     <width>89</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight</set>
   </property>
   <property name="name" stdset="0">
    <cstring>txtSSDUtil</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>300</y>
     <width>111</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>SSD Utilization</string>
   </property>
   <property name="name" stdset="0">
    <cstring>txtLabelSSDUtil</cstring>
   </property>
  </widget>
 <widget class="QTextEdit" name="txtDebug">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>360</y>
     <width>460</width>
     <height>80</height>
    </rect>
   </property>
   <property name="mouseTracking">
    <bool>false</bool>
   </property>
   <property name="name" stdset="0">
    <cstring>txtDebug</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>340</y>
     <width>120</width>
     <height>20</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Debug Status</string>
   </property>
   <property name="name" stdset="0">
    <cstring>txtLabelDebug</cstring>
   </property>
  </widget>
  <widget class="QLineEdit" name="txtTime">
   <property name="geometry">
    <rect>
     <x>130</x>
     <y>300</y>
     <width>89</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>0</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignRight</set>
   </property>
   <property name="name" stdset="0">
    <cstring>txtTime</cstring>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>300</y>
     <width>50</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Time</string>
   </property>
   <property name="name" stdset="0">
    <cstring>txtLabelTime</cstring>
   </property>
  </widget>
  <widget class="QPushButton" name="btnSave">
   <property name="geometry">
    <rect>
     <x>490</x>
     <y>90</y>
     <width>100</width>
     <height>50</height>
    </rect>
   </property>
   <property name="text">
    <string>Save to File</string>
   </property>
   <property name="name" stdset="0">
    <cstring>btnSave</cstring>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
